generator/template/Emoji.java

```diff
@@ -44,27 +62,52 @@
   }
 
   @NonNull @Override public Drawable getDrawable(final Context context) {
```

Comment:
Are you sure? LruCache is thread-safe. As far as I can see, the worst case if the method's called concurrently is that the callers return different bitmaps backed by the same strip, which is also possible on master. Is there a case I'm missing?

Reply:
Ah, did not know `LruCache` is thread safe. Nevertheless I think we should synchronize here, since it feels more clean, especially when we have the `destroy` method back. What do you think?

Reply:
If we can avoid synchronizing we should.

Reply:
I looked at this again, you are right, synchronizing `loadStrip` is enough.